### PR TITLE
Spring Boot web task.

### PR DESCRIPTION
**Pull request by TkachukTara**

### CODE REVIEW

File: src/main/java/book/store/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package book.store.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    private Long id;
```
Comment (Rommelua) [inline src/main/java/book/store/dto/CreateBookRequestDto.java]:
```suggestion
```

File: src/main/java/book/store/service/impl/BookServiceImpl.java [hunk 2]
```diff
@@ -1,28 +1,41 @@
...
     private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
 
-    @Autowired
-    public BookServiceImpl(BookRepository bookRepository) {
-        this.bookRepository = bookRepository;
+    @Override
+    public BookDto save(CreateBookRequestDto createBookRequestDto) {
+        Book book = bookMapper.toModel(createBookRequestDto);
+        book.setIsbn("Isbn" + new Random().nextInt(1000));
```
Comment (Rommelua) [inline src/main/java/book/store/service/impl/BookServiceImpl.java]: we should have it in CreateBookRequestDto 
```suggestion
```

File: src/main/java/book/store/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")
+public class BookController {
+
```
Comment (MentorMA) [inline src/main/java/book/store/controller/BookController.java]: Why do you need an empty line at start of class

File: src/main/java/book/store/dto/CreateBookRequestDto.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    private String title;
+    private String author;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+    private String isbn = "Isbn" + new Random().nextInt(1000);
```
Comment (MentorMA) [inline src/main/java/book/store/dto/CreateBookRequestDto.java]:
```suggestion
    private String isbn;
```

Comment (MentorMA) [reply src/main/java/book/store/dto/CreateBookRequestDto.java]: Why do you init it here?

**Review by MentorMA — CHANGES_REQUESTED** [review]

**Review by koval-kn — APPROVED** [review]

**Pull request merged**